services/src/contexts/migrations/migration_0016_merge_providers.sql

```diff
@@ -61,11 +61,20 @@ DROP TYPE "ProDataProviderDefinition";
 
 -- user_sessions
 
-DELETE FROM sessions;
-
 ALTER TABLE sessions ADD COLUMN
-user_id uuid REFERENCES users (id) ON DELETE CASCADE NOT NULL;
-ALTER TABLE sessions ADD COLUMN created timestamp with time zone NOT NULL;
-ALTER TABLE sessions ADD COLUMN valid_until timestamp with time zone NOT NULL;
+user_id uuid REFERENCES users (id) ON DELETE CASCADE;
```

Comment:
the user_session table had 

> PRIMARY KEY (user_id, session_id)

so maybe we should do that in the session_table as well?

Reply:
Nein, Anfragen werden über die Session ID gestellt, für joins gibt es auch der User ID einen Fremdschlüssel plus Index